src/v2/components/FireList.jsx

```diff
@@ -93,21 +93,45 @@ export default function FireList(props) {
   const handleVoteForFire = useCallback((index, vote) => {
     if (index > -1 && index < nFires) {
       const fire = fires[index]
-      const {cameraID, timestamp} = fire
-      const isRealFire = vote === 'yes'
-      const isUndo = vote === 'undo'
-      const endpoint = isUndo ? '/api/undoVoteFire' : '/api/voteFire'
-
-      query.post(endpoint, {cameraID, isRealFire, timestamp}).then(() => {
-        if (isUndo) {
-          delete fire.voted
-        } else {
-          fire.voted = isRealFire
-        }
-
-        updateFires(indexOfOldFires > -1)
-      }) // TODO: Implement error handling. Allow errors to go uncaught for now
-         // asy they will be logged to the Console.
+      const promises = [fire]
+        .concat(Object.keys(fire._anglesByKey).map((k) => firesByKey[k]))
+        .reduce((a, f) => {
+          const {cameraID, timestamp} = f
+
+          // Vote only for `fire` and overlapping angles older than `fire`.
+          if (timestamp > fire.timestamp) {
```

Comment:
Should use sortId instead of timestamp.  Perhaps timestamp should be renamed to displayTimestamp or similar...

Reply:
`sortId` it is. Sorry about that. I’m not sure renaming will fully eliminate the confusion (for me at least). But if `sortId` is renamed `timestamp`, I’d support renaming the current `timestamp` as `displayTimestamp`.